src/wallet/rpc/spend.cpp

```diff
@@ -488,13 +487,13 @@ static std::vector<RPCArg> FundTxDoc(bool solving_data = true)
     return args;
 }
 
-void FundTransaction(CWallet& wallet, CMutableTransaction& tx, CAmount& fee_out, int& change_position, const UniValue& options, CCoinControl& coinControl, bool override_min_fee)
+std::optional<CreatedTransactionResult> FundTransaction(CWallet& wallet, const CMutableTransaction& tx, const UniValue& options, CCoinControl& coinControl, bool override_min_fee)
```

Comment:
Since this can never return a nullopt (errors throw), maybe just return `CreatedTransactionResult`?